velox/dwio/dwrf/test/TestColumnReader.cpp

```diff
@@ -3781,6 +3781,184 @@ TEST_P(TestColumnReader, testTimestamp) {
   }
 }
 
+TEST_P(TestColumnReader, testDecimal64) {
+  // set getEncoding
+  proto::ColumnEncoding directEncoding;
+  directEncoding.set_kind(proto::ColumnEncoding_Kind_DIRECT);
+  EXPECT_CALL(streams_, getEncodingProxy(_))
+      .WillRepeatedly(Return(&directEncoding));
+
+  // set getStream
+  EXPECT_CALL(streams_, getStreamProxy(_, proto::Stream_Kind_ROW_INDEX, false))
+      .WillRepeatedly(Return(nullptr));
+  EXPECT_CALL(streams_, getStreamProxy(_, proto::Stream_Kind_PRESENT, false))
+      .WillRepeatedly(Return(nullptr));
+  // col_0's Data Stream
+  char numBuffer[65];
+  for (int i = 0; i < 65; ++i) {
+    if (i < 32) {
+      numBuffer[i] = static_cast<char>(0x3f - 2 * i);
+    } else {
+      numBuffer[i] = static_cast<char>(2 * (i - 32));
+    }
+  }
+  EXPECT_CALL(streams_, getStreamProxy(1, proto::Stream_Kind_DATA, true))
+      .WillRepeatedly(Return(new SeekableArrayInputStream(
```

Comment:
ASAN is complaining about memory leaking.  Can we make the test leak free?

Reply:
Done.